### DISCUSSION TITLE
read .ssh/config to configure `wezterm connect`

### DISCUSSION THREAD

**Discussion by ho-oto** [wezterm/wezterm#1731] [Q&A] [answered]
What is the best practice to use `Host` defined in `.ssh/config` as ssh_domains? Can I use some functions like [`SSHConfig::parse_str`](https://docs.rs/ssh_config/latest/ssh_config/) in `wezterm.lua`?

**Comment by wez** [comment, marked as answer]
I just added this function; it should show up in the nightly builds within an hour of posting this comment.
https://wezfurlong.org/wezterm/config/lua/wezterm/enumerate_ssh_hosts.html
